composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -1,73 +1,89 @@
+// External dependencies.
 import React from "react";
 import styled from "styled-components";
 import PropTypes from "prop-types";
+import uniqueId from "lodash/uniqueId";
+import { __ } from "@wordpress/i18n";
 
+// Internal dependencies.
 import colors from "../../../../style-guide/colors.json";
 
-let KeywordField = styled.input`
-	margin-right: 0.5em;
-	border-color: ${ props => props.borderColor };
+const errorColor = colors.$color_red;
+
+const KeywordInputContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 1em 0;
 `;
 
-const ErrorText = styled.div`
+const KeywordFieldLabel = styled.label`
 	font-size: 1em;
-	color: ${ colors.$color_red };
-	margin: 1em 0;
-	min-height: 1.8em;
+	font-weight: bold;
+	margin-bottom: 0.5em;
 `;
 
-const ErrorMessage = "Are you trying to use multiple keywords? You should add them separately below.";
+const KeywordField = styled.input`
+	border-color: ${ colors.$color_grey_light };
+	border-style: solid;
+	border-width: 3px;
+	padding: 0.75em;
+	font-size: 1.1em;
```

Comment:
why 1.1em? in the standalone example this is based on the root font size which is 16px so it's 17.6px. In WordPress it will probably be 13 x 1.1 = 14.3 pixels.